core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -683,7 +682,9 @@ void core_init_mmu_regs(void)
 
 enum core_mmu_fault core_mmu_get_fault_type(uint32_t fsr)
 {
-	assert(!(fsr & FSR_LPAE));
+	if (fsr & FSR_LPAE)
```

Comment:
An `assert()` should be enough here, this is more of a configuration issue than a run-time issue (even if it's detected at run-time).